test/test.js

```diff
@@ -173,6 +194,30 @@ describe('bot', function() {
       assert.equal(subscribe('guild1', 'user2', 'SAMPLE GAME'), 'Sample game');
       assert.equal(subscribe('guild1', 'user2', 'fuzzy game'), 'FUZZY GAME');
     });
+
+    it ('should be able to list subscribed games', function() {
+      Bot.create(MockConfig(), MOCK_HOOKS);
+
+      assert.equal(get_subscriptions('guild1', 'user2').length, 0);
+
+      assert.equal(subscribe('guild1', 'user2', 'Sample game'), 'Sample game');
+      assert.equal(get_subscriptions('guild1', 'user2').length, 1);
+      assert.equal(get_subscriptions('guild1', 'user2')[0], 'Sample game');
+
+      // Multiple games are listed alphabetically.
+      assert.equal(subscribe('guild1', 'user2', 'Other game'), 'Other game');
+      assert.equal(get_subscriptions('guild1', 'user2').length, 2);
+      assert.equal(get_subscriptions('guild1', 'user2')[0], 'Other game');
+      assert.equal(get_subscriptions('guild1', 'user2')[1], 'Sample game');
+
+      // There is also special formatting for 3 games, so test that too.
+      assert.equal(subscribe('guild1', 'user2', 'FUZZY GAME'), 'FUZZY GAME');
+      assert.equal(get_subscriptions('guild1', 'user2').length, 3);
+      assert.equal(get_subscriptions('guild1', 'user2')[0], 'FUZZY GAME');
+      assert.equal(get_subscriptions('guild1', 'user2')[1], 'Other game');
+      assert.equal(get_subscriptions('guild1', 'user2')[2], 'Sample game');
```

Comment:
For this you can use assert.deepEqual(get_subscriptions('guild1', 'user2'), ['FUZZY GAME', 'Other game', 'Sample game')